modules/widgets/twitter-timeline.php

```diff
@@ -80,9 +80,11 @@ public function widget( $args, $instance ) {
 
 		echo $args['before_widget'];
 
-		if ( isset( $instance['title'] ) ) {
+		$title = isset( $instance['title'] ) ? $instance['title'] : '';
+		$title = apply_filters( 'widget_title', $title );
+		if ( ! empty( $title ) ) {
 			/** This filter is documented in core/src/wp-includes/default-widgets.php */
```

Comment:
Could you move the docblock so it remains on the line above the filter?

Thanks!